app/MMB/src/scenes/tickets/boardingPasses/appleWallet/AppleWallet.js

```diff
@@ -12,6 +12,7 @@ import type { AppleWallet as Pkpasses } from './__generated__/AppleWallet.graphq
 
 type Props = {|
   +data: Pkpasses,
+  +legId: ?string,
```

Comment:
Shall we start renaming these with `segmentId`? Since you are already using the (correct) word _segment_ for your context.

Reply:
Yes, this is already a bit confusing, since we fetch it like `leg.id` but we use it like segmentId later 🤔 

I will rename it.